dsms/core/configuration.py

```diff
@@ -74,10 +74,31 @@ class Configuration(BaseSettings):
         description="JWT bearer token for connecting to the DSMS instance",
     )
 
+    reauthenticate: bool = Field(
```

Comment:
You mean auth reauthentication, right? it's not so clear from the var name and the description. Perhaps `enable_auto_reauth` could be more self-descriptive var name.

Reply:
Has been applied.